docs/pages/operations-guide.md

```diff
@@ -280,32 +280,13 @@ A Task generally maps to a process. A Pod is a collection of Tasks that share an
 
 ## Virtual networks
 
-The SDK allows pods to join virtual networks, with the `dcos` virtual network available by defualt. You can specify that a pod should join the virtual network by adding the following to your service spec YAML:
-
-```yaml
-pods:
-  pod-on-virtual-network:
-    count: {{COUNT}}
-    # join the 'dcos' virtual network
-    networks:
-      dcos:
-    tasks:
-      ...
-  pod-on-host:
-    count: {{COUNT}}
-    tasks:
-      ...
-```
+The SDK allows pods to join virtual networks, with the `dcos` virtual network available by defualt. You can specify that a pod should join the virtual network by using the `networks` keyword in your YAML definition. Refer to [Developers Guide](developer-guide.md) for more information about how to define virtual networks in your service.
```

Comment:
Should also talk about labels here. Maybe add:
...in your YAML definition. *SDK pods can interface with CNI plug-ins by adding labels. These are free-form key-value pairs that are passed in the format key:value,key2:value2.*Refer to